File: app/src/main/java/com/fq/demo/xposed/feature/MainActivity.java
package com.fq.demo.xposed.feature;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import com.fq.demo.xposed.R;

/**
 * MainActivity2019/3/12 10:08 AM
 *
 * @desc :
 */
public class MainActivity extends AppCompatActivity {


    private TextView textView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        textView = findViewById(R.id.m_tv_label);
        textView.setText("Hello init");
    }
}

File: app/src/main/java/com/fq/demo/xposed/feature/MainActivity.kt
package com.fq.demo.xposed.feature

import android.os.Build
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.fq.demo.xposed.R
import kotlinx.android.synthetic.main.activity_main.*

/**
 * MainActivity2019/3/12 3:23 PM
 * @desc :
 *
 */
class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        m_tv_label.text = "Hello init"

        m_tv_models.text = Build.MODEL
        m_tv_version.text = Build.VERSION.RELEASE
    }

    fun getTvLabel() = m_tv_label


}